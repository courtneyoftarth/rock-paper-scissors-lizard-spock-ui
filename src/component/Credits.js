import React from 'react';

const Credits = () => (
    <div>
        <h2>Credits</h2>
        Pictures taken from
        <ul>
            <li><a href="https://www.istockphoto.com/vector/friendly-green-iguana-in-flat-style-vector-gm862153016-142888289">Lizard</a></li>
            <li><a href="https://www.pinclipart.com/pindetail/ohwhox_star-trek-spock-fallout-star-trek-spock-fallout/">Spock</a></li>
        </ul>
    </div>
);

export default Credits;