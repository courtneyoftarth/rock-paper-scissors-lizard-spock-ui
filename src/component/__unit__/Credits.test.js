import React from 'react';
import { render } from '@testing-library/react';
import Credits from '../Credits';

test('Credits renders successfully', () => {
    const { container } = render(<Credits />);
    expect(container).toMatchSnapshot();
});
