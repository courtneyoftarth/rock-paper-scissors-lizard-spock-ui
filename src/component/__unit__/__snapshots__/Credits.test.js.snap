// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credits renders successfully 1`] = `
<div>
  <div>
    <h2>
      Credits
    </h2>
    Pictures taken from
    <ul>
      <li>
        <a
          href="https://www.istockphoto.com/vector/friendly-green-iguana-in-flat-style-vector-gm862153016-142888289"
        >
          lizard
        </a>
      </li>
      <li>
        <a
          href="https://www.pinclipart.com/pindetail/Txmmx_counseling-clipart-student-meeting-paper-black-and-white/"
        >
          paper
        </a>
      </li>
      <li>
        <a
          href="https://clipartart.com/categories/rock-clipart-free.html"
        >
          rock
        </a>
      </li>
      <li>
        <a
          href="http://www.clipartpanda.com/clipart_images/scissors-clipart-5181114"
        >
          scissors
        </a>
      </li>
      <li>
        <a
          href="https://www.pinclipart.com/pindetail/ohwhox_star-trek-spock-fallout-star-trek-spock-fallout/"
        >
          spock
        </a>
      </li>
    </ul>
  </div>
</div>
`;
